applications/Tokenguard.md

```diff
@@ -193,7 +193,8 @@ Tokenguard addresses the need of following audiences:
 
 
 Tokenguard's public features and data are already being used by community users and projects like DotInsights ([https://dotinsights.subwallet.app/polkadot-report-q2-2023-en/](https://dotinsights.subwallet.app/polkadot-report-q2-2023-en/)):
-<img src="https://imagizer.imageshack.com/img922/508/rymrP6.png" width="800" border="0" />
+<img src="https://imagizer.imageshack.com/img922/508/rymrP6.png" width="800" border="0">
```

Comment:
I think this would trip the markdown parser that builds our website. Is it possible that we added the `/` after the application was merged? If so, could you re-add it?

Reply:
pinging @caaaml

Reply:
Hey @semuelle, the file is changed - can you please check if it works fine now? :) Thanks!